docs/nGQL-tutorial.md

```diff
@@ -0,0 +1,165 @@
+# Nebula Graph Query Language Tutorial
+
+### Statements
+
+`nGQL` is a declarative, textual query language like SQL, so its queries are both easy to write, and to read.
+
+nGQL key words are case-insensitive but we recommend them written in all caps for easy reading. Based on the released Nebula v0.1.0 and corresponding Docker mirror, we have created a simple graph with 4 vertices and 3 edges to help you get a quick understanding of nGQL.
+
+### Graph space administration
+
+Graph spaces are physically isolated like the database in MySQL.
+
+| | CREATE | DROP | USE | DESCRIBE | SHOW |
+|---| --- | --- | ----- | -------- | ---- |
+| SPACE | √ | √  | √    | v0.2     | √    |
+
+Create space with CREATE, drop space with DROP, choose which space to use with USE, list available spaces with SHOW. DESCRIBE will be released in v0.2.
+
+Following are some examples:
+
+* List all the spaces available
+
+```
+SHOW SPACES
+```
+
+* Drop a space
+
+```
+DROP SPACE myspace_test
+```
+**Note:** DROP SPACE deletes all data in the current version and recovery is not supported yet.
+
+* Create a space
+
+```
+CREATE SPACE myspace_test(partition_num=10, replica_factor=1)
```

Comment:
currently we have support `CREATE SPACE ` without options

Reply:
This doc is based on v0.1. It's better to release a minor version, e.g. v0.1.3, to announce  more features,

Reply:
I think a doc release note will be better if we have more docs in the future.